sympy/tensor/index_methods.py

```diff
@@ -24,6 +23,7 @@ class IndexConformanceException(Exception):
     pass
 
 
+@deprecated(useinstead='EinsteinSum', issue=9284)
 def _remove_repeated(inds):
```

Comment:
I removed the deprecation of ``_remove_repeated`` (not committed yet).

As for e.g. `_get_indices_Mul`, IMO leaving the deprecation decorator in is the best option. Adding the deprecation warning to the expected output seems awkward to me.

Disagree?